internal/auth/configuration.go

```diff
@@ -35,6 +38,9 @@ import (
 // PROVIDER_*_CLIENT_SECRET
 // PROVIDER_*_SCOPE
 //
+// PROVIDER_*_AZURE_TENANT
+// PROVIDER_*_AZURE_PROMPT
```

Comment:
This snuck in somehow, not sure how critical it is to keep this isolated since this PR is the main blocker for Azure, but I can remove if needed.